src/Resources/config/services.yml

```diff
@@ -36,6 +36,15 @@ services:
             - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest, priority: 30 }
             - { name: kernel.event_listener, event: kernel.finish_request, method: onKernelFinishRequest, priority: -254 }
 
+    contao.listener.exception:
+        class: Contao\CoreBundle\EventListener\ExceptionListener
+        tags:
+            - { name: kernel.event_listener, event: kernel.exception, method: onKernelException }
+        arguments:
+            - "%contao.pretty_error_screens%"
+            - "@templating"
```

Comment:
As far as I know, you can use `@twig` directly in Symfony 2.7 (see http://symfony.com/blog/new-in-symfony-2-7-twig-as-a-first-class-citizen#using-twig-without-the-templating-sub-system)